No such file or directory, scandir '/home/runner/conda_pkgs_dir'

Thank you for an awesome GitHub workflow / conda integration, has been using it for a little while and it is super helpful.

Starting today (15/05/2019) I noticed a bunch of CIs failed in ` Run goanpeca/setup-miniconda@v1` with: 

```
Append to "/home/runner/.xonshrc":
 
# ----------------------------------------------------------------------------
# Conda Setup Action: Basic configuration
set -eo pipefail
# Conda Setup Action: Custom activation
conda activate my_env
# ---------------------------------------------------------------------------- 

Append to "/home/runner/.config/powershell/profile.ps1":
 
# ----------------------------------------------------------------------------
# Conda Setup Action: Custom activation
conda activate my_env
# ---------------------------------------------------------------------------- 


# Removing uncompressed packages to trim down cache folder...
##[error]ENOENT: no such file or directory, scandir '/home/runner/conda_pkgs_dir'
```

An example workflow where I can repro:

```
  build:
    runs-on: ubuntu-latest

...

    - uses: goanpeca/setup-miniconda@v1
      with:
        activate-environment: my_env

```

It appears to be related to the new release 1.3.0 which was pushed just yesterday; when I locked version to `goanpeca/setup-miniconda@v1.2.0` the builds resumed successfully.

--- a/src/setup-conda.ts
+++ b/src/setup-conda.ts
@@ -734,17 +734,19 @@ async function setupMiniconda(
       if (!result["ok"]) return result;
     }
 
-    consoleLog("Removing uncompressed packages to trim down cache folder...");
-    let fullPath: string;
-    for (let folder_or_file of fs.readdirSync(cacheFolder)) {
-      fullPath = path.join(cacheFolder, folder_or_file);
-      if (
-        fs.existsSync(fullPath) &&
-        fs.lstatSync(fullPath).isDirectory() &&
-        folder_or_file != "cache"
-      ) {
-        core.info(`Removing "${fullPath}"`);
-        await io.rmRF(fullPath);
+    if (fs.existsSync(cacheFolder) && fs.lstatSync(cacheFolder).isDirectory()) {
+      consoleLog("Removing uncompressed packages to trim down cache folder...");
+      let fullPath: string;
+      for (let folder_or_file of fs.readdirSync(cacheFolder)) {
+        fullPath = path.join(cacheFolder, folder_or_file);
+        if (
+          fs.existsSync(fullPath) &&
+          fs.lstatSync(fullPath).isDirectory() &&
+          folder_or_file != "cache"
+        ) {
+          core.info(`Removing "${fullPath}"`);
+          await io.rmRF(fullPath);
+        }
       }
     }
   } catch (err) {

